plugins/admin/pkg/command/registry/registry.go

```diff
@@ -47,15 +47,26 @@ type registryCred struct {
 // Auths is a map of docker credentials indexed by server url
 type Auths map[string]registryCred
 
+type registryAdminParams struct {
+	AdminParams    *pkg.AdminParams
+	Namespace      string
+	ServiceAccount string
+}
```

Comment:
Is registryAdminParams added for the Namespace/ServiceAccount to be shared by add.go and remove.go?

Reply:
Following kn client style, it may not be a good idea to set persist flags in `Registry` for sharing. So now they are specified in `Add` and `Remove` commands separately.